spokestack/asr/speech_recognizer.py

```diff
@@ -0,0 +1,89 @@
+"""
+This module contains the recognizer for cloud based ASR
+"""
+
+import numpy as np  # type: ignore
+
+from spokestack.asr.cloud_client import CloudClient
+from spokestack.context import SpeechContext
+
+
+class CloudSpeechRecognizer:
+    """ speech recognizer """
+
+    def __init__(
+        self,
+        spokestack_id: str = "",
+        spokestack_secret: str = "",
+        language: str = "en",
+        sample_rate: int = 16000,
+        frame_width: int = 10,
+    ) -> None:
+        self._client_id: str = spokestack_id
+        self._secret: str = spokestack_secret
+        self._language: str = language
+        self._sample_rate: int = sample_rate
+        self._frame_width: int = frame_width
+        self.max_idle_count = 5000 / frame_width
+        self._client: CloudClient = CloudClient(
+            self._client_id, self._secret, self._language, sample_rate=self._sample_rate
+        )
+        self._is_active = False
+        self._idle_count: int = 0
+        self._final = True
+        self._context: SpeechContext
```

Comment:
we don't want to store the context, use the value that is passed to the callable